.github/workflows/parachain.yml

```diff
@@ -160,11 +171,31 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: setup rust toolchain
         run: rustup show
-      - name: snowbridge integration
+      - name: snowbridge runtime tests
         run: >
           cargo test 
+          --manifest-path parachain/Cargo.toml
+          -p snowbridge-runtime-tests
+          -- --nocapture
+
+  integration-tests:
+    needs: check
+    runs-on: snowbridge-runner
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
+      - uses: arduino/setup-protoc@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: setup rust toolchain
+        run: rustup show
+      - name: snowbridge integration tests
+        run: >
+          RUST_LOG=xcm=trace cargo test 
           --manifest-path polkadot-sdk/Cargo.toml
-          -p bridge-hub-rococo-integration-tests
+          -p bridge-hub-rococo-integration-tests snowbridge
```

Comment:
What package is named `snowbridge`? I don't recall any package like that?

Reply:
Actually it's not a package name, the mod name to filter snowbrige tests only.